ui/k-info-panel/show_circuit.kytos

```diff
@@ -755,10 +758,10 @@ module.exports = {
         name: this.basics["Name"]["value"],
         enable: this.flags["Enabled"]["value"],
         dynamic_backup_path: this.flags["Dynamic backup path"]["value"],                
-        sb_priority: parseInt(this.others["Southbound priority"]["value"]),
-        bandwidth: this.others["Bandwidth"]["value"],
-        queue_id: this.others["Queue"]["value"],
-        service_level: this.others["Service level"]["value"],
+        sb_priority: parseInt(this.others["Southbound priority"]["value"]) || 0,
```

Comment:
@rmotitsuki, 

Can you revert `sb_priority` and `queue_id` to not default to 0? If they're not set they should be `null`, otherwise it would have implications on OpenFlow values that would be set. For instance, queue_id: 0 would result in `set_queue` action